configcatclient/rolloutevaluator.py

```diff
@@ -39,31 +39,31 @@ def evaluate(self, key, user, default_value, default_variation_id, settings):
         setting_descriptor = settings.get(key)
 
         if setting_descriptor is None:
-            error = 'Evaluating get_value(\'{}\') failed. Value not found for key \'{}\' ' \
-                    'Returning default_value: [{}]. ' \
-                    'Here are the available keys: {}'.format(key, key, str(default_value),
-                                                             ', '.join(list(settings)))
-
-            self.log.error(error)
-            return default_value, default_variation_id, None, None, error
+            error = 'Failed to evaluate setting \'%s\' (the key was not found in config JSON). ' \
+                    'Returning the `%s` parameter that you specified in your application: \'%s\'. ' \
+                    'Available keys: %s.'
+            errorArgs = (key, 'default_value', str(default_value), ', '.join(list(settings)))
```

Comment:
Available keys should be surrounded with single quotes like `'Key1', 'Key2', 'Key3'`